determined-ai/determined
# feat: add new Delete Workspace Namespace API Bindings CLI and API endpoint

## Ticket




## Description


Created new API and CLI endpoints to delete workspace namespace bindings and fixed a bug in validateRequestClusterName

## Test Plan

1. Start a Kubernetes cluster.
2. Create a workspace and create namespace bindings.
3. try to delete the binding by using the command `det w bindings delete <workspace-name> --cluster-name <cluster-name>`
4. Make sure you get the success message.
5. Use the same command and delete again and make sure you get an error message saying you tried to delete a default binding

## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
